options.go

```diff
@@ -18,7 +18,10 @@ type Options struct {
 // bidiText converts a bi-directional text logically to visually.
 func bidiText(in string) (out string, err error) {
 	p := bidi.Paragraph{}
-	p.SetString(in)
+	_, err = p.SetString(in)
+	if err != nil {
+		return
+	}
```

Comment:
We don't really have to handle this one, as it will only return an error if options are passed to it, but we are not passing any options. So, it's safe!

But yeah, thanks for the fix.